hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueFactory.kt

```diff
@@ -19,7 +19,7 @@ import uk.gov.justice.hmpps.sqs.HmppsSqsProperties.QueueConfig
 class HmppsQueueFactory(
   private val context: ConfigurableApplicationContext,
   private val healthContributorRegistry: HmppsHealthContributorRegistry,
-  private val sqsClientFactory: SqsClientFactory,
+  private val sqsClientFactory: SqsClientFactory
```

Comment:
this looks the wrong way round - comma shouldn't have been removed here

Reply:
argh yes - thought I checked this !